tests/lib/tools/fs-state

```diff
@@ -1,38 +1,53 @@
 #!/bin/bash
 
+show_help() {
+	echo "usage: fs-state wait-for-file [filename] [iterations] [sleep-time]"
+	echo "usage: fs-state ensure-dir-exists [direname]"
+	echo "usage: fs-state ensure-dir-exists-backup-real [direname]"
```

Comment:
Could this be called something like `mock-file` and be paired with `restore-file`.